.github/workflows/build.yml

```
@@ -19,7 +21,25 @@ jobs:
         with:
           nix_path: nixpkgs=./nixpkgs.nix
 
-      - run: echo "build --host_platform=@rules_haskell//haskell/platforms:linux_x86_64_nixpkgs" > .bazelrc.local
+      - name: Configure
+        run: |
+          mkdir -p ~/repo-cache ~/disk-cache
+          echo "build --host_platform=@rules_haskell//haskell/platforms:linux_x86_64_nixpkgs" > .bazelrc.local
```

```suggestion
          echo "build --host_platform=@rules_haskell//haskell/platforms:linux_x86_64_nixpkgs $BAZEL_ARGS" > .bazelrc.local
```
I don't see otherwise how the arguments are communicated to bazel.

---

I have updated it ...